implementations/rust/ockam/ockam_command/src/status/static/long_about.txt

```diff
@@ -0,0 +1,8 @@
+The status command displays the current status of your machine, including information such as:
+- the current version of Ockam in use
+- the controller (Orchestrator) and project version
+- your spaces and the users associated with them
+- all cryptographic identities stored in your vault and their enrollment status with the Ockam Orchestrator
+- the status of any Nodes, associated routes and protocols, listeners and services, and more
+
+  NB: To see the list of workers running in a node, run `ockam worker list --at <NAME_OF_NODE>`
```

Comment:
This last line can be omitted. Workers are a lower-level primitive that most users won't need to interact with directly.